### ISSUE TITLE
Improve storefront examples if logic is used

### ISSUE THREAD

**Issue by anotherMatze** [porsche-design-system/porsche-design-system#1637]
Scope
---
Components which are controlled by the user should be better explained with logic examples (tabs-bar, banner, more?).

While doing so, the accessibility example of tabs-bar should be improved and the bug where the `hidden` attribute stays displayed in the documentation should be fixed.

Out of Scope
---
List topics that are not part of this task.

Notes  
---
Share any further information that is relevant for processing.

Acceptance criteria
---
- [ ] Criteria

Subtasks
---
- [ ] Task
